src/Nethermind/Nethermind.Network.Discovery/NettyDiscoveryHandler.cs

```diff
@@ -80,27 +80,26 @@ public override void ChannelReadComplete(IChannelHandlerContext context)
 
     public async void SendMsg(DiscoveryMsg discoveryMsg)
     {
-        byte[] msgBytes;
+        IByteBuffer msgBuffer = Unpooled.Buffer(64);
```

Comment:
Why 64?
If there is a reason then please add a comment.

Reply:
64 is the default length used in ZeroSerialize when we do not know the length of message, so I used something similar here. Or we can standardize the GetLength interface for all the messages and then can use that here.

Reply:
Please add a comment to the code - similar to the one here :)